ReactUI/src/ui/components/materialUI/converter/FormVideoConverter.jsx

```diff
@@ -0,0 +1,601 @@
+												onChange={(e) =>
+													setFrameScale(
+														e.target.value
+													)
+												}
+											>
+												<MenuItem value="">
+													<em>-</em>
+												</MenuItem>
+												<MenuItem value={"854"}>
+													854
+												</MenuItem>
+												<MenuItem value={"640"}>
+													640
+												</MenuItem>
+												<MenuItem value={"520"}>
+													520
+												</MenuItem>
+												<MenuItem value={"426"}>
+													426
+												</MenuItem>
+												<MenuItem value={"360"}>
+													360
+												</MenuItem>
+												<MenuItem value={"256"}>
+													256
+												</MenuItem>
+											</Select>
+										</FormControl>
+									</Grid>
+									<Grid item xs>
+										<FormControlLabel
+											control={
+												<Checkbox
+													checked={obtainFrames}
+													onChange={(e) => {
+														setObtainFrames(
+															e.target.checked
+														);
+													}}
+													name="obtainFrames"
+													color="primary"
+												/>
+											}
+											label="Obtain frames"
+										/>
+									</Grid>
+								</Grid>
+							</CardContent>
+						</Card>
+						<br></br>
+						<Card className={classes.paper}>
+							<CardHeader subheader="Obtain Audio" />
+							<CardContent>
+								<Grid container spacing={6}>
+									<Grid item xs>
+										<FormControl
+											variant="outlined"
+											className={classes.formControl}
+											fullWidth
+										>
+											<InputLabel id="demo-simple-select-outlined-label">
+												Audio format
+											</InputLabel>
+											<Select
+												labelId="demo-simple-select-outlined-label"
+												id="demo-simple-select-outlined"
+												value={extractAudioFormat}
+												onChange={(e) =>
+													setExtractAudioFormat(
+														e.target.value
+													)
+												}
+												label="Audio format"
+											>
+												<MenuItem value="">
+													<em>-</em>
+												</MenuItem>
+												<MenuItem value={".mp3"}>
+													.mp3
+												</MenuItem>
+												<MenuItem value={".mp2"}>
+													.mp2
+												</MenuItem>
+												<MenuItem value={".wma"}>
+													.wma
+												</MenuItem>
+											</Select>
+										</FormControl>
+									</Grid>
+									<Grid item xs>
+										<FormControlLabel
+											control={
+												<Checkbox
+													checked={obtainAudio}
+													onChange={(e) => {
+														setObtainAudio(
+															e.target.checked
+														);
+													}}
+													name="obtainAudio"
+													color="primary"
+												/>
+											}
+											label="Obtain Audio"
+										/>
+									</Grid>
+								</Grid>
+							</CardContent>
+						</Card>
+					</Grid>
+					<Grid item xs={7}>
+						<Card className={classes.paper}>
```

Comment:
Here is the change that I put in FormImgConverter, sorry.
![image](https://user-images.githubusercontent.com/71795880/128649722-093ed438-d634-48dd-963e-1c36a2e60c01.png)
But also apply this idea in the other comment.

Reply:
Same here, this issue will be written in a ticket